input/europe/write_reactors.py

```diff
@@ -0,0 +1,43 @@
+import sys
```

Comment:
Does this file need to exist in two places in the repository? I'd strongly discourage it. I'd recommend making it's capabilities general. There is no reason it needs to be in the same directory as the files it operates on.